Neos.Utility.ObjectHandling/Classes/TYPO3/Flow/Reflection/ObjectAccess.php

```diff
@@ -97,13 +97,10 @@ protected static function getPropertyInternal($subject, $propertyName, $forceDir
     {
         if ($subject === null) {
             return null;
-        }
-        if (is_array($subject)) {
-            if (array_key_exists($propertyName, $subject)) {
-                $propertyExists = true;
-                return $subject[$propertyName];
-            }
-            return null;
+        } elseif (is_array($subject)) {
+            $propertySet = isset($subject[$propertyName]);
+            $propertyExists = $propertySet || array_key_exists($propertyName, $subject);
```

Comment:
You're probably right. This is way too much micro-optimization. With PHP 5.6 `array_key_exists()` is about three times slower than `isset()` but with PHP 7 it's way closer ;)